### ISSUE TITLE
Next build failing

### ISSUE THREAD

**Issue by fadeev** [zeta-chain/docs#447]
Example:

https://github.com/zeta-chain/docs/pull/446

https://vercel.com/zeta-chain/docs-v2/3RhExhB864PuUzYkHg6QoKWB5Z2J?filter=errors

```
Error: Command "NODE_OPTIONS=--max-old-space-size=6144 next build" exited with SIGKILL
```

**Comment by fadeev** [comment]
If you re-run it a few times, it may work.

**Comment by fadeev** [comment]
@GMaiolo @juansalvatore the fix in https://github.com/zeta-chain/docs/pull/451 didn't resolve the issue.

**Comment by fadeev** [comment]
For example, see https://github.com/zeta-chain/docs/pull/443

**Comment by GMaiolo** [comment]
I see deployments passing, if you got a link to a new one with the implemented changes in #451 please link it
